commands/run.go

```diff
@@ -181,6 +196,24 @@ func newJudgeReport() JudgeReport {
 	return &ConsoleJudgeReport{Stats: []CaseStatus{}}
 }
 
+func timedExecution(cmd *exec.Cmd, timeOut float64) (int8, time.Duration, error) {
```

Comment:
I suggest adding a doc block to describe how time execution work, for future reference.